project_task_pull_request_state/__manifest__.py

```diff
@@ -10,9 +8,12 @@
     "license": "AGPL-3",
     "application": False,
     "installable": True,
-    "depends": ["project_task_pull_request"],
+    "depends": [
+        "project",
```

Comment:
project_task_pull_request already depends on project. No need to add it explicitly